### ISSUE TITLE
About 'combined' dataset

### ISSUE THREAD

**Issue by Alyssa14411** [allenai/ai2thor-rearrangement#19]
Hi, I found the `combined` data size is 4340, which is not the sum of the four datasets above.
For example, for the `train & train_unseen`, the floorplan is 1-20, 200-220, 300-320, and 400-420, each contains 50 scenes. But in the `combined`, these 120 floorplans only contain 15 or 30 scenes each.
So what is the relationship between combined and the four `train, train_unseen, val, test` datasets?
Thanks a lot!

**Comment by mattdeitke** [comment]
Hi @Alyssa14411,

* **Train** is used for training agents, utilizing the first 20 rooms of each category. FloorPlan1-20 are kitchens, FloorPlan200-220 are living rooms, FloorPlan300-320 are bedrooms, FloorPlan400-420 are bathrooms.
* **Train unseen** is a validation set, used to determine how well the agent can generalize to novel rearrangement tasks, inside of the same rooms it has trained on (i.e., `FloorPlanX[1-20]`).
* **Val** is a validation set to see how well the agent can generalize to novel rooms that are not seen during training. These include 20 new rooms, with 5 new rooms of each room category (i.e., FloorPlan21-25 for kitchens, FloorPlan221-225 for living rooms, FloorPlan321-325 for bedrooms, and FloorPlan421-425 for bathrooms).
* **Test** is a testing set, similar to the validation set, but should be used much less frequently such that models do not overfit to it. This also includes 20 new rooms, with 5 new rooms of each room category (i.e., FloorPlan26-30 for kitchens, FloorPlan226-230 for living rooms, FloorPlan326-330 for bedrooms, and FloorPlan426-430 for bathrooms). The testing set is also used to determine rankings on the leaderboard.

More information about scenes in AI2-THOR can also be found on [their documentation's Scenes page](https://ai2thor.allenai.org/ithor/documentation/scenes/) :)

**Comment by Alyssa14411** [comment]
Got it, may thanks ;)
